### PR TITLE
Create basic code

### PR DESCRIPTION
**Pull request by AbdealiJK**

### CODE REVIEW

**Comment by AbdealiJK** [comment]
@tushar-rishav @sils1297 I'd like both of you to review this please.
As I'm not sure what I'm doing myself :P (I did this a while back ... and I don't remember much)

**Comment by AbdealiJK** [comment]
I've revamped the code to not use Grunt but has all the other stuff which you'd like - review again please.
@sils1297 gitmate needed here now

File: .coafile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[js]
+files = app/**/*.js
+bears = LineLengthBear, SpaceConsistencyBear, JSHintBear
+jshint_config = ./.jshintrc
```
Comment (sils1297) [inline .coafile]: we should really add a way to autogenerate this

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "4.1"
+
+notifications:
+  email: false
```
Comment (sils1297) [inline .travis.yml]: thank you so much for setting this

**Comment by tushar-rishav** [comment]
@rultor merge

**Comment by sils1297** [comment]
`'cid' file is absent, container wasn't started correctly` is a rultor bug, if that occurs just try again.

**Comment by tushar-rishav** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge :@

**Comment by sils1297** [comment]
@rultor merge

**Comment by AbdealiJK** [comment]
@rultor merge

**Comment by AbdealiJK** [comment]
Im fairly sure that this is happening because there is NO rultor.yml file in the `master` branch yet.
So, Im doing this manually.
I remember that rultor uses the master's rultor.yml

That's probably what it means by "build automation tool" not found

**Comment by AbdealiJK** [comment]
Done. Next time onwards we can use rultor :)

**Comment by tushar-rishav** [comment]
@AbdealiJK We probably need the updates in `gh-pages` branch. Is there any way rultor could merge with `gh-pages` instead of looking for default master branch? Otherwise we'll have to update `gh-pages` every time master is updated. `gh-pages` is essential for demo.

**Comment by sils1297** [comment]
Should be possible

**Pull request closed without merging**